web/main/colors.css

```diff
@@ -83,12 +89,14 @@
 .is-slow-theme {
   @media (prefers-color-scheme: light) {
     --land-color: color-mix(in oklab, var(--sand-100), var(--sand-200));
+    --theme-color: oklch(94.15% 0.01 82.4);
```

Comment:
Adding extra color variable is dangerous, because we may forget to update it.

What do you think if we will use:

```js
window.getComputedStyle(document.body).getPropertyValue('background') 
```

`getComputedStyle` returns styles after all CSS magic, so it will be just `rgb()` value, no custom properties, etc.

Reply:
This works indeed (I changed it to get `background-color` though), good idea